csrc/ir/internal_base_nodes.h

```diff
@@ -554,18 +554,26 @@ class TensorDomain : public Val {
     return no_bcast_domain_;
   }
 
+  // The input logical domain. The root domain of a consumer should equal the
+  // rfactor domain of its producer.
```

Comment:
"... ignoring IterDomains of type Reduction."

Reply:
Fixed.